src/external/token/ERC20IssuanceUpgradeable_v1.sol

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+// External Interfaces
+import {IERC20Issuance_v1} from "@ex/token/IERC20Issuance_v1.sol";
+
+// External Dependencies
+import {
+    ERC20Upgradeable,
+    ERC20CappedUpgradeable
+} from "@oz-up/token/ERC20/extensions/ERC20CappedUpgradeable.sol";
+import {OwnableUpgradeable} from "@oz-up/access/OwnableUpgradeable.sol";
+
+/**
+ * @title   Inverter ERC20 Issuance Token (Upgradeable)
+ *
+ * @notice  An upgradeable ERC20 token implementation with a maximum supply cap
+ *          and controlled minting/burning through a whitelist permission system.
+ *
+ * @dev     This contract implements the following key functionalities:
+ *          - Role-based access control through a minter whitelist
+ *          - Controlled token issuance (minting) by authorized addresses only
+ *          - Controlled token redemption (burning) by authorized addresses only
+ *          - Hard cap on total token supply to prevent inflation
+ *          - Support for contract upgrades through OpenZeppelin's
+ *            upgradeable pattern
+ *
+ * @custom:setup    This contract requires the following MANDATORY setup steps:
+ *
+ *                  Set Minter:
+ *                     - Purpose: The contract needs a minter to handle token
+ *                                minting and burning operations. Without this
+ *                                permission, the workflow cannot mint or burn
+ *                                tokens.
+ *                     - How:     The owner of the contract must call the
+ *                                setMinter function to authorize the Funding
+ *                                Manager of the workflow
+ *                     - Example: token.setMinter(fundingManagerAddress, true);
+ *
+ * @custom:security-contact security@inverter.network
+ *                          In case of any concerns or findings, please refer to
+ *                          our Security Policy at security.inverter.network or
+ *                          email us directly!
+ *
+ * @custom:version  v1.0.0
+ *
+ * @custom:standard-version v1.0.0
+ *
+ * @author Inverter Network
+ */
+contract ERC20IssuanceUpgradeable_v1 is
+    IERC20Issuance_v1,
+    ERC20CappedUpgradeable,
+    OwnableUpgradeable
+{
+    // State Variables
+    //------------------------------------------------------------------------------
```

Comment:
I think this is too long + missing space at the start. (not just here but throughout the contract i think)